lms/djangoapps/email_marketing/signals.py

```diff
@@ -90,15 +90,15 @@ def add_email_marketing_cookies(sender, response=None, user=None,
 
 
 @receiver(REGISTER_USER)
-def email_marketing_register_user(sender, user=None, profile=None,
+def email_marketing_register_user(sender, user=None, registration=None,
```

Comment:
@schenedx Yes it will work, because we don't pass these arguments directly to celery task, we pass them to `_create_sailthru_user_vars()` to create `sailthru_vars`.